raiwidgets/raiwidgets/dashboard.py

```diff
@@ -76,7 +77,10 @@ def __init__(self, *,
 
         html = self.load_index()
         print(f'{dashboard_type} started at {self._service.env.base_url}')
-        display(HTML(html), metadata=dict(isolated=True))
+        if "DATABRICKS_RUNTIME_VERSION" in os.environ:
```

Comment:
Shouldn't this be a function in the flask wrapper? I think it's actually there already, we just don't use it yet.

Reply:
done, used display function